core/server/data/schema/schema.js

```diff
@@ -56,7 +56,8 @@ module.exports = {
         twitter_image: {type: 'string', maxlength: 2000, nullable: true},
         twitter_title: {type: 'string', maxlength: 300, nullable: true},
         twitter_description: {type: 'string', maxlength: 500, nullable: true},
-        custom_template: {type: 'string', maxlength: 100, nullable: true}
+        custom_template: {type: 'string', maxlength: 100, nullable: true},
+        canonical: {type: 'text', maxlength: 2000, nullable: true, validations: {isEmptyOrURL: true}}
```

Comment:
Was the naming discussed? IMO the more specific the better e.g. `canonical_url`.

Reply:
Good catch. Will check!